torchvision/transforms/v2/_misc.py

```diff
@@ -249,12 +252,29 @@ def __init__(self, dtype: Union[torch.dtype, Dict[Type, Optional[torch.dtype]]])
                 "in case a `datapoints.Image` or `datapoints.Video` is present in the input."
             )
         self.dtype = dtype
+        self.scale = scale
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        dtype = self.dtype[type(inpt)]
+        if type(inpt) in self.dtype:
+            dtype = self.dtype[type(inpt)]
```

Comment:
Not sure if relevant in this PR: should we also do an `isinstance` check here afterwards? Might be useful in the context of custom datapoints. Same applies to the `fill` parameter on some other transforms. Maybe we can unify the functionality behind these "type-dict parameters"?

Reply:
You mean ensuring that the resulting `dtype` is an actual dtype (like torch.float32)? I assume that users would get a decent error message down the road anyway, at least when `.to()` gets called?